files/en-us/web/javascript/reference/global_objects/number/index.html

```diff
@@ -119,8 +119,8 @@ <h3 id="Integer_range_for_Number">Integer range for Number</h3>
 
 <p>The following example shows the minimum and maximum integer values that can be represented as <code>Number</code> object. (More details on this are described in the ECMAScript standard, chapter <em><a href="https://tc39.github.io/ecma262/#sec-ecmascript-language-types-number-type">6.1.6 The Number Type</a>.</em>)</p>
 
-<pre class="brush: js notranslate">const biggestInt  = Number.MAX_SAFE_INTEGER  //  (<code>2<sup>53</sup> - 1</code>) =&gt;  9007199254740991
-const smallestInt = Number.MIN_SAFE_INTEGER  // -(<code>2<sup>53</sup> - 1</code>) =&gt; -9007199254740991</pre>
+<pre class="brush: js notranslate">const biggestInt  = Number.MAX_SAFE_INTEGER  //  (<code>2**53 - 1</code>) =&gt;  9007199254740991
+const smallestInt = Number.MIN_SAFE_INTEGER  // -(<code>2**53 - 1</code>) =&gt; -9007199254740991</pre>
```

Comment:
```suggestion
<pre class="brush: js notranslate">const biggestInt  = Number.MAX_SAFE_INTEGER  //  (<code>2^53 - 1</code>) =&gt;  9007199254740991
const smallestInt = Number.MIN_SAFE_INTEGER  // -(<code>2^53 - 1</code>) =&gt; -9007199254740991</pre>
```
I think the caret in the more common and familiar way to indicate superscript; see https://pages.uoregon.edu/ncp/Courses/MathInPlainTextEmail.html

> Use "^" for superscripts: 2^6, e^3, etc. ("**" instead of "^" is also OK.)

So while that says the double-asterisk syntax is "OK", it suggests we should prefer the caret.